src/app/components/preferences/qemu/add-qemu-vm-template/add-qemu-vm-template.component.ts

```diff
@@ -109,17 +107,14 @@ export class AddQemuVmTemplateComponent implements OnInit {
       });
 
 
-      this.qemuService.getBinaries(this.controller).subscribe((qemuBinaries: QemuBinary[]) => {
-        this.qemuBinaries = qemuBinaries;
-        if (this.qemuBinaries[0]) this.selectedBinary = this.qemuBinaries[0];
-      });
+      
 
       this.qemuService.getImages(this.controller).subscribe((qemuImages: QemuImage[]) => {
         this.qemuImages = qemuImages;
       });
 
       this.selectPlatform = this.configurationService.getPlatform();
-      this.selectedPlatform = this.selectPlatform[0];
+      // this.selectedPlatform = this.selectPlatform[0];
```

Comment:
Do we need to comment this line?

Reply:
I think we need `this.selectedPlatform = this.selectPlatform[0];` because x86_64 must be the default when creating a new Qemu template